content/news/2021-03-01-week-13-progress.md

```diff
@@ -0,0 +1,21 @@
+title: "About my Outreachy internship with Mixxx DJ: Week 13 progress"
+authors: Aanyu Deborah Oduman
+tags: personal, outreachy, internship, manual
+date: 2021-03-01 19:33:00
+
+_This is the thirteeth weekly progress report of my Outreachy internship with Mixxx DJ._
+
+This week:
+
+- Following last week’s progress, I finished making the edits to the video, and now, I’m only waiting to get feedback before I add in any animations or effects. I hope to upload it this week (since I’m writing this blog in the next week). 🤞🏽
+- I have been working on [issue #117](https://github.com/mixxxdj/manual/issues/117) – updating the screenshots of the [2.3 manual](https://manual.mixxx.org/2.3/en/index.html) from the Deere to Latenight skin. I hope to make a PR this week.
+
+And… this is the last week of the Outreachy internship program. This week’s blog is not as long as the previous ones because I have not documented what I will be doing next week. This is because I have not planned out all of the details yet, but of course, I will be making improvements to the manual – next week I will be updating the screenshots in the 2.3 manual from the Deere to the Latenight skin. I hope to check this item off the checklist in this [issue](https://github.com/mixxxdj/manual/issues/117). And then, of course, I will continue making tutorials for the [Mixxx Youtube channel](https://www.youtube.com/watch?v=ADh_svLuU7g&feature=youtu.be) and working on the cookbook/ DJing techniques chapter.
```

Comment:
```suggestion
And… this is the last week of the Outreachy internship program. This week’s blog is not as long as the previous ones because I have not documented what I will be doing next week. This is because I have not planned out all of the details yet, but of course, I will be making improvements to the manual – next week I will be updating the screenshots in the 2.3 manual from the Deere to the Latenight skin. I hope to check this item off the checklist in this [issue](https://github.com/mixxxdj/manual/issues/117). And then, of course, I will continue making tutorials for the [Mixxx Youtube channel](https://www.youtube.com/channel/UC1t9_tylS7XPBIJIpXhXQmQ) and working on the cookbook/ DJing techniques chapter.
```